publication-doi-commons/src/test/java/no/unit/nva/publication/doi/dto/PublicationDateTest.java

```diff
@@ -18,27 +18,12 @@
 
     @Test
     void testJsonNodeConstructor() {
-        var actual = new PublicationDate(getPublicationWithDate());
+        var actual = PublicationDate.fromJsonNode(getPublicationWithDate());
         assertThat(actual.getYear(), is(equalTo("1999")));
         assertThat(actual.getMonth(), is(equalTo("07")));
         assertThat(actual.getDay(), is(equalTo("09")));
     }
 
-    @Test
```

Comment:
Removed test with no clear expectations. The is test is also irrelevant now as PublicationDate cannot be created unless there is a year.